### ISSUE TITLE
Certain files weren't present.

### ISSUE THREAD

**Issue by Fatih20** [thiagoojack/packettracer-fedora#29]
Hi! I was trying to install the Packet Tracer and during the extraction process of the files, I got this message.
```
./opt/pt/translations/qtxmlpatterns_uk.qm
./opt/pt/translations/qtxmlpatterns_zh_CN.qm
./opt/pt/translations/qtxmlpatterns_zh_TW.qm
./usr/
./usr/share/
./usr/share/icons/
./usr/share/icons/gnome/
./usr/share/icons/gnome/48x48/
./usr/share/icons/gnome/48x48/mimetypes/
./usr/share/icons/gnome/48x48/mimetypes/pka.png
./usr/share/icons/gnome/48x48/mimetypes/pks.png
./usr/share/icons/gnome/48x48/mimetypes/pksz.png
./usr/share/icons/gnome/48x48/mimetypes/pkt.png
./usr/share/icons/gnome/48x48/mimetypes/pkz.png
./usr/share/mime/
./usr/share/mime/packages/
./usr/share/mime/packages/cisco-pka.xml
./usr/share/mime/packages/cisco-pks.xml
./usr/share/mime/packages/cisco-pksz.xml
./usr/share/mime/packages/cisco-pkt.xml
./usr/share/mime/packages/cisco-pkz.xml
sed: can't read /usr/share/applications/cisco-pt.desktop: No such file or directory
gtk-update-icon-cache: Cache file created successfully.
touch: cannot touch '/root/.config/mimeapps.list': No such file or directory
/usr/bin/xdg-mime: line 873: /root/.config/mimeapps.list.new: No such file or directory
```

And then the script terminate itself. I am using the `8.2.1` version of the Packet Tracer from the given URL. The operating system I was using is this :
![image](https://github.com/thiagoojack/packettracer-fedora/assets/51940383/2c9a46eb-abb1-4775-b843-c41a5d60d5b2)

Any assistance would be appreciated!

**Comment by thiagoojack** [comment]
Hi! Are you running the script using root account? If yes, this is your problem.

**Comment by Fatih20** [comment]
Yep I did! Alright, after I retried the script without using `sudo` it worked. But the previous line regarding unable to find the `.desktop` file was caused by the packet tracker itself having the name `cisco-pt821.desktop` and `cisco-ptsa821.desktop`.  I had to do a bit of a trick of removing it out of the folder (because it gets deleted every time the script runs), renaming it, and then moving it back when the script is running (but after it finished deleting the `.desktop` files).
